### DISCUSSION TITLE
Opening Images in Windows Apps from IG

### DISCUSSION THREAD

**Discussion by aickau** [d2phap/ImageGlass#746] [General]
Hi,

I would like to open images from IG in Polarr. Is there a workaround how to do to configure IG to do that?

Image Glass provides the oppurtunity to open images in 3rd party image editing programs like Photoshop and so on. This works perfectly with traditional programs but not with image editing tools that are deployed as "Windows Applications". It would be great if IG could also do that in some way.

**Comment by d2phap** [comment]
There is a tutorial how to open UWP app from command:
https://www.addictivetips.com/windows-tips/open-uwp-apps-from-command-line-windows-10/
It takes quite along to get the command after many steps.

The other ways is if the UWP app register protocol, but what I check in Polarr, there is no protocol in AppxManifest.xml file.
![image](https://user-images.githubusercontent.com/3154213/79638222-19331d80-81b7-11ea-8afb-59cfb0f2995d.png)

I don't know any solutions to pass the file to UWP Polarr for now.

✅ However, you can **hold CTRL and drag the viewing image to Polarr** window. It works by this way.
